packages/next/src/experimental/testmode/fetch.ts

```diff
@@ -88,7 +88,8 @@ export async function handleFetch(
 ): Promise<Response> {
   const testInfo = getTestReqInfo(request, reader)
   if (!testInfo) {
-    throw new Error(`No test info for ${request.method} ${request.url}`)
+    // Passthrough non-test requests.
+    return originalFetch(request)
```

Comment:
This change allows Next.js to return the original response instead of throwing an error